tasks.py

```diff
@@ -0,0 +1,77 @@
+### Using the element path strategy when locating application elements.
+
+import io
+import random
+
+from robocorp.tasks import task, setup
+from robocorp.tasks._protocols import ITask
+from robocorp import log
+from robocorp import windows
+
+APP_EXECUTABLE = "calc.exe"
+APP_LOCATOR = "name:Calculator"
+PATH_TO_MAINVIEW = "path:2|3|2"
+PATH_TO_RESULT = f"{PATH_TO_MAINVIEW}|2|1" # path:2|3|2|2|1
```

Comment:
Using hard-coded paths for the calculator is tricky as it changes across windows versions, so, this is bound to fail depending on the user machine...

I did a test-case for it in:
https://github.com/robocorp/robo/blob/master/windows/tests/robocorp_windows_tests/test_robocorp_windows_calc.py

For instance, to get the result I'm using:

`window.find("name:Result or id:CalculatorResults", timeout=0)`

To click buttons, it's usually easier to just send the keys when possible:

i.e.:

`window.send_keys(keys="96+4=")`